admin/spec/spec_helper.rb

```diff
@@ -90,7 +90,6 @@
 
   config.include ViewComponent::TestHelpers, type: :component
   config.include ViewComponent::SystemTestHelpers, type: :component
-  config.include SolidusAdmin::ComponentHelpers, type: :component
```

Comment:
I can see situations in which this could be useful in the future, I guess we can also get it back from git history, but would also be fine to keep it, thoughts?

Reply:
I prefer to get it back if we need it in the future. If the current codebase doesn't use it, there's no reason for it to be there.